src/components/checkbox/checkbox.template.tsx

```
@@ -50,52 +50,37 @@ export const CheckboxTemplate: FunctionalComponent<CheckboxTemplateProps> = (
+                class="mdc-checkbox__native-control" // required by MDC to work
+                id={props.id}
+                checked={props.checked}
+                disabled={props.disabled || props.readonly}
+                required={props.required}
+                onChange={props.onChange}
+                aria-controls={props.helperTextId}
+                aria-describedby={props.helperTextId}
+                {...inputProps}
+            />
+            <div class="box">
+                <svg viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)